microsoft-365-dke/docker/Dockerfile

```diff
@@ -22,21 +22,19 @@ RUN curl -LO# https://github.com/stedolan/jq/releases/download/jq-1.6/jq-linux64
     mv jq /usr/bin    
 
 RUN mkdir /root/data
-COPY data /root/data
+COPY . /root/data
```

Comment:
Why did you cancel the /data folder and moved all scripts to the main folder ?
This will copy Dockerfile and build.sh also into the /data folder in the container which is not needed